tests/docker/docker-compose.yml

```diff
@@ -6,6 +6,27 @@ networks:
     driver: bridge
 
 services:
+  minio:
+    command: server /data
+    container_name: minio-s3
+    environment:
+      MINIO_DOMAIN: minio-s3
+      MINIO_REGION_NAME: panda-region
+      MINIO_ROOT_PASSWORD: panda-secret
+      MINIO_ROOT_USER: panda-user
+    expose:
+    - '9000'
+    healthcheck:
+      interval: 30s
+      retries: '3'
+      test:
+      - CMD
+      - curl
+      - -f
+      - http://localhost:9000/minio/health/live
+      timeout: 20s
+    image: minio/minio:RELEASE.2021-03-26T00-00-41Z
+
   n1: &node
     image: vectorized/redpanda-test-node
     privileged: true
```

Comment:
seems that the errors are coming from minio not starting by the time the tests run. One thing that might help is to declare all the nodes as dependent of the `minio` service, something like:

```suggestion
    privileged: true
    depends_on: minio
      condition: service_healthy
```

Reply:
docker compose complains that such syntax is not supported, I ended up doing this:
```
priveleged: true
depends_on:
- minio
```
also, documentation says that 'condition' doesn't work with 'depends_on' anymore (in Ver 3)